script/post_split.rb

```diff
@@ -0,0 +1,92 @@
+new_subject = 'test subject'
+reply_user = 'Throne3d'
+reply_id = 1024 # we split *after* this reply_id
+
+Post.transaction do
+  user = User.find_by(username: reply_user)
+  abort("needs user") unless user
+  reply = Reply.find_by(user_id: user.id, id: reply_id)
+  abort("couldn't find reply") unless reply
+  old_post = reply.post
+  puts "splitting post #{old_post.id}: #{old_post.subject}, after reply #{reply_id}"
+
+  first_reply = old_post.replies.where('reply_order > ?', reply.reply_order).ordered.first
+  other_replies = old_post.replies.where('reply_order > ?', first_reply.reply_order).ordered
```

Comment:
Why _are_ we splitting after the given id, that seems to add unnecessary complexity?